packages/@aws-cdk/aws-msk-alpha/lib/cluster.ts

```diff
@@ -103,6 +103,7 @@ export interface ClusterProps {
    * @default - 1000 GiB EBS volume
    */
   readonly ebsStorageInfo?: EbsStorageInfo;
+  
```

Comment:
Looks like a typo newline and whitespace character. Is breaking build.
```suggestion
```

Reply:
Again, the build is breaking due to a hidden tab, not that the new line exists here